meshapp/src/profile/profile-experience.tsx

```diff
@@ -0,0 +1,302 @@
+
+  };
+
+  //TODO: Receive and store experience
+  function saveOccupation(text: string, index: number) {
+    occupations[index] = text;
+  }
+
+  function saveOrganization(text: string, index: number) {
+    organizations[index] = text;
+  }
+
+  function saveDescription(text: string, index: number) {
+    descriptions[index] = text;
+  }
+
+  /**
+   * Displays experience header with edit/save icon.
+   */
+  const ExperienceHeader = () => {
+    return (
+      <Grid item container gap="10px" alignItems="center" mb="20px">
+        <Typography
+          variant="h1"
+          sx={{
+            fontSize: "22px",
+          }}
+        >
+          Experience
+        </Typography>
+        {editMode ? (
+          <SaveIcon
+            color="primary"
+            onClick={handleSaveClick}
+            sx={{
+              "&:hover": {
+                color: "#0A6B57",
+              },
+            }}
+          />
+        ) : (
+          <EditIcon
+            color="disabled"
+            onClick={handleEditClick}
+            sx={{
+              "&:hover": {
+                color: "#0b7d66",
+              },
+            }}
+          />
+        )}
+      </Grid>
+    );
+  };
+
+  /**
+   * Displays the user's experience in edit mode.
+   *
+   * @param props - Properties of the component
+   * @param {string} props.title - Title of user's occupation
+   * @param {string} props.organization - Name of business/org the user affiliates with
+   * @param {string} props.description - Description of user's occupation
+   * @param {boolean} props.expand - Expand mode indicator for user's occupation
+   * @param {numberg} props.index- Index of user's occupation
+   */
+  const EditExperience = (props: {
+    title: string;
+    organization: string;
+    description: string;
+    expand: boolean;
+    index: number;
+  }) => {
+    return (
+      <Grid container alignItems="center" gap="10px">
+        <RemoveButton />
+        <Grid item {...(editMode ? { xs: 11.57 } : { xs: 12 })}>
+          <Accordion expanded={props.expand}>
+            <AccordionSummary
+              expandIcon={
+                <ExpandMoreIcon
+                  onClick={() => handleExpandClick(props.index)}
+                />
+              }
+            >
+              <Grid container gap="20px">
+                <Grid xs={5.8}>
+                  <ProfileTextField
+                    variant="standard"
+                    label="Occupation"
+                    placeholder=""
+                    text={props.title}
+                    charLimit={50}
+                    handleSave={(text) => saveOccupation(text, props.index)}
+                  />
+                </Grid>
+                <Grid xs={5.8}>
+                  <ProfileTextField
+                    variant="standard"
+                    label="Organization"
+                    placeholder=""
+                    text={props.organization}
+                    charLimit={50}
```

Comment:
This seems to be common across all the `ProfileTextField`s. I think this should be a constant.